airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSourceOperations.java

```diff
@@ -261,6 +278,26 @@ protected void putBoolean(final ObjectNode node, final String columnName, final
     node.put(columnName, resultSet.getString(index).equalsIgnoreCase("t"));
   }
 
+  protected void putFloat8(final ObjectNode node, final String columnName, final ResultSet resultSet, final int index)
+      throws SQLException {
+    final BigDecimal bigDecimal = DataTypeUtils.returnNullIfInvalid(() -> resultSet.getBigDecimal(index));
```

Comment:
why does this need to be BigDecimal, if the column is just an 8-byte float? Is that not equivalent to a java `double`?

Reply:
Fixed. Now it's `double`